subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/accessors/AccessorsClassPath.kt

```diff
@@ -472,8 +472,8 @@ internal
 fun TypedProjectSchema.toCacheKeyString(): String =
     (cacheKeyPartsFor(extensions)
         + cacheKeyPartsFor(conventions)
-        //+ cacheKeyPartsFor(tasks) // TODO:accessors - add missing test case
-        //+ cacheKeyPartsFor(containerElements) // TODO:accessors - add missing test case
+        // + cacheKeyPartsFor(tasks) // TODO:accessors - add missing test case
```

Comment:
:x: this change doesn't seem relevant to this PR.

Reply:
ktlint was upgraded and this is because of a new rule
it is a good thing I think